### ISSUE TITLE
Conversion of Word to asciidoc

### ISSUE THREAD

**Issue by marjacks76** [itc-wgtools/cPP-Tools#1]
We need to work out how to best advise folks on the conversion of existing Word .docx content in to a more open format. This issue will track explorations in this area. I've initially started looking at Pandoc for the conversion of the NDcPP and ND SD v2.1 documents using the following the guidance at https://asciidoctor.org/docs/migrating-from-msword/

More details as I explore further.

**Comment by atsecsysad** [comment]
Just a reminder for the group heret: AsciiDoctor is not the same thing as asciidoc. They're two different implementations, although the instructions in the post are not connected to this difference which is great. 

Here's a page on the AsciiDoctor site illustrating the differences for reference: 
https://asciidoctor.org/docs/asciidoc-asciidoctor-diffs/

**Comment by woodbe** [comment]
I was looking at this briefly and checked the Google Docs path that was listed and generally the answer is no, I don't think that will work.

I think that following the preparation steps for getting it ready and then converting using Pandoc is the only automated path. I did a lot of copy and pasting creating the templates, and while it is annoying, it works (though you need to be careful of getting binary codes embedded as I did in a few docs causing them to show up as binary files instead of text).

**Comment by marjacks76** [comment]
Apologies for not being so active on this issue as I'd have liked, the day-job and other Network iTC activities have been keeping me busy! 

The Network iTC has just published v2.2 of the cPP and SD and as the lead technical editor, I once again suffered from the pain of being tied to MS Word based documents. With this pain fresh in mind, I took some time over the holiday period to experiment with the conversion process of the Network cPP and SD using pandoc. I plan to document the experience which I'll then share with the group.

**Comment by woodbe** [comment]
So I played around with this using my Security Target and found that the process will get all the text (I didn't have images in this file so I don't know how they will come over), but I can say I don't like the tables (as they leave it looking like a spreadsheet with everything on one row in one line, and I think it is cleaner to have the table rows grouped but on separate lines for each cell).

So here are the differences I found from the instructions on the page above:

1. Download the latest pandoc to make sure you have support for the latest Word versions
2. Save the file as a normal DOCX file, not as the strict XML thing specified as it will not work if you do this (it give an error about not being able to read the file)
3. If you are going to extract images make sure you have the folder specified before you run the command (it won't create them for you)
4. you want to specify asciidoctor, not asciidoc for the output (there are differences, so specify asciidoctor as that is what we are using)

When I did this, while it pulled the tables, it didn't properly label them, so I'm not really sure why I needed to move all the table captions to the top of the tables (though it is easier to find after the fact).

Also, make sure you use the ">" to redirect the output to the new file, otherwise it will give you an error.

In the end, I'm not sure how beneficial this is overall vs just copy and pasting into the editor. There are certain advantages, but at the same time all the formatting has been lost, so you will need to go back through both files anyway to re-insert the proper bold/italics for the selections/assignments and anything else. I'm not sure this is easier after-the-fact or during the process directly.
